### PR TITLE
Hide input box when text input is not active

### PR DESCRIPTION
**Pull request by benrucker**
## Old behavior:

https://user-images.githubusercontent.com/12519846/209749487-8d31ad2d-6b81-4b5e-a423-538b213108e8.mp4

## New behavior:

https://user-images.githubusercontent.com/12519846/209749598-1b90ba6a-02bd-42dd-8423-fc987f97e0d2.mp4

### CODE REVIEW

**Comment by benrucker** [comment]
also removed some unused code + changed the terminal shadow

File: components/whyd/2022/Terminal/UserInputArea.vue [hunk 1]
```diff
@@ -36,6 +36,7 @@
       </button>
     </div>
     <input
+      v-show="isTextInputModeActive"
```
Comment (ecfidler) [inline components/whyd/2022/Terminal/UserInputArea.vue:39]: why is this necessary / what issue is this preventing?
